bolt/lib/Passes/LongJmp.cpp

```diff
@@ -354,9 +354,12 @@ uint64_t LongJmpPass::tentativeLayoutRelocMode(
   for (BinaryFunction *Func : SortedFunctions) {
     if (!BC.shouldEmit(*Func)) {
       HotAddresses[Func] = Func->getAddress();
-      continue;
+      // Don't perform any tentative address estimation of a function's cold
+      // layout if it won't be emitted, unless we are ignoring a large number of
```

Comment:
I think If we won't skip working on these functions we would end up with setting new address on 376 line which would be the wrong address for this function

Reply:
Thanks @yota9 for your review. This should be addressed now. Can you please confirm?